internal/projection/process/event_enricher_test.go

```diff
@@ -86,9 +86,11 @@ var _ = Describe("Process message", func() {
 
 			keys := make([]string, 0)
 			for _, m := range connectivityMajorityGroupsList {
-				conn, ok := m.(map[string]interface{})
+				var conn map[string]interface{}
```

Comment:
A bit silly from the linter requiring this to be declared, we can clearly see what type it is without declaring. Also the `ok` variable is not declared, not sure why it's not complaining about that?

Reply:
Its because of shadowing, it is complaining that "ok" was declared twice